docs/getting_started.md

```diff
@@ -228,6 +227,20 @@ Examples:
 ```python
 convert_ply('./test.ply', './test.bin')
 ```
+If you have point clouds in other format (`off`, `obj`, etc.), you can use trimesh to convert them into `ply`.
+```python
```

Comment:
code block should have an empty line above and below it.

Reply:
Fix.